core/src/main/scala/caliban/schema/Step.scala

```diff
@@ -1,7 +1,8 @@
 package caliban.schema
 
 import caliban.CalibanError.ExecutionError
-import caliban.Value.NullValue
+import caliban.ResponseValue.ObjectValue
```

Comment:
Unused imports ?

Reply:
Removed. By the way shall we add a linter to check for unused imports and run it during CI only? It's pretty easy to miss them unfortunately

Reply:
Yeah that'd be nice.

Reply:
Ok I'll add it in a separate PR 👍